package.json

```diff
@@ -233,15 +233,13 @@
           ],
           "description": "Have VSCode download the language server automatically (MacOS/Windows only for Swift-MesonLSP)"
         },
-        "mesonbuild.setupCppTools": {
-          "type": "boolean",
-          "default": true,
-          "markdownDescription": "Setup the [Microsoft C/C++ extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools) to work with Meson"
-        },
-        "mesonbuild.setupRustAnalyzer": {
-          "type": "boolean",
+        "mesonbuild.modifySettings": {
+          "type": [
+            "boolean",
+            "array"
+          ],
```

Comment:
Include the other part of the JSON schema snippet I sent you on Matrix.

Reply:
done